streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -482,7 +482,8 @@ public class StreamsConfig extends AbstractConfig {
     public static final String BUILT_IN_METRICS_VERSION_CONFIG = "built.in.metrics.version";
     private static final String BUILT_IN_METRICS_VERSION_DOC = "Version of the built-in metrics to use.";
 
-    /** {@code cache.max.bytes.buffering} */
+    /** {@code cache.max.bytes.buffering}
+     * @deprecated since 3.4.0 Use cache.max.bytes instead with the cache.size metric at the DEBUG level. */
```

Comment:
```suggestion
     * @deprecated since 3.4.0 Use {@link #CACHE_MAX_BYTES_CONFIG "cache.max.bytes"} instead. */
```

Reply:
Not sure if I understand the reference to `cache.size` metric? Can you elaborate?

Reply:
I believe I had gleaned that from a release blog but I can't find it now... happy to commit the suggestion